### PR TITLE
enhanced method record entries

### PR DESCRIPTION
**Pull request by mohamed-barakat**

### CODE REVIEW

File: MonoidalCategories/gap/ClosedMonoidalCategoriesMethodRecord.gi [hunk 1]
```diff
@@ -261,6 +261,9 @@ IsomorphismFromInternalHomToDual := rec(
 IsomorphismFromDualToInternalHom := rec(
   filter_list := [ "category", "object" ],
   io_type := [ [ "a" ], [ "d", "i" ] ],
+  output_source_getter_string := "DualOnObjects( cat, a )",
+  output_range_getter_string := "InternalHomOnObjects( cat, a, TensorUnit( cat ) )",
+  with_given_object_position := "both",
```
Comment (zickgraf) [inline MonoidalCategories/gap/ClosedMonoidalCategoriesMethodRecord.gi:266]: This line will probably cause errors.

File: MonoidalCategories/gap/CoclosedMonoidalCategoriesMethodRecord.gi [hunk 2]
```diff
@@ -261,6 +261,9 @@ IsomorphismFromCoDualToInternalCoHom := rec(
 IsomorphismFromInternalCoHomToCoDual := rec(
   filter_list := [ "category", "object" ],
   io_type := [ [ "a" ], [ "i", "d" ] ],
+  output_source_getter_string := "InternalCoHomOnObjects( cat, TensorUnit( cat ), a )",
+  output_range_getter_string := "CoDualOnObjects( cat, a )",
+  with_given_object_position := "both",
```
Comment (zickgraf) [inline MonoidalCategories/gap/CoclosedMonoidalCategoriesMethodRecord.gi:266]: This line too.

**Review by zickgraf — CHANGES_REQUESTED** [review]

File: MonoidalCategories/gap/ClosedMonoidalCategoriesMethodRecord.gi (hunk 1 above)
Comment (mohamed-barakat) [reply MonoidalCategories/gap/ClosedMonoidalCategoriesMethodRecord.gi:266]: Yes, it did :)

**Review by zickgraf — CHANGES_REQUESTED** [review]
Is there a reason why you have not touched `IsomorphismFromInternalHomToDual` and `IsomorphismFromCoDualToInternalCoHom`? I think those should also be changed for consistency.

**Comment by TKuh** [comment]
Since we need to work anyway on the isomorphisms because of #929, what is the status here?

**Comment by zickgraf** [comment]
@mohamed-barakat Since you are the original author of the PR and since you have originally requested a review from me, I will now merge this.

**Pull request merged**